scripts/spack/packages/serac/package.py

```diff
@@ -217,6 +227,11 @@ class Serac(CachedCMakePackage, CudaPackage):
     # Conflicts
     #
 
+    conflicts("+openmp", when="+rocm")
```

Comment:
Can we get a comment on this conflict? Unlike cuda/rocm, this should work.

Reply:
im not sure why this is needed, i just copied this from axom. I could test it out and see what happens when i try 

https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/axom/package.py#L215

Reply:
haven't actually tried till now, but it results in a conflict anyway. since we propagate the openmp variant to axom, its conflict is in effect as well.